config/helm/repos/stable/index.yaml

```diff
@@ -1,5 +1,303 @@
 apiVersion: v1
 entries:
+  dynatrace-oneagent-operator:
```

Comment:
do we want the old operator here ? 
(I know we kinda want parity with the helm-chart repo, but this feels icky to me 😅 )

Reply:
Removed